src/app/api/password-reset/route.ts

```diff
@@ -24,27 +24,21 @@ export const POST = withAppRouterHighlight(async (request: NextRequest) => {
       .executeTakeFirst();
 
     if (!storedUser) {
-      return new Response(
-        "A password reset email will be sent if you have a Flojoy Cloud account. Please check your inbox :)",
-        {
-          status: 200,
-        },
-      );
+      return new Response("User does not exist!", {
```

Comment:
A password reset email will be sent if you have a Flojoy Cloud account. Please check your inbox :)

Reply:
Restore to this old message for the same reason mentioned above